packages/world/src/modules/coldmethods/ColdMethodsSystem.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Could we just add these to the RegistrationSystem/Module? (Or even bundle all external World methods in the same CoreModule/System)? The advantage of that would be that the address of this CoreSystem is more likely to be warmed up in one transaction, so subsequent calls are cheaper

Reply:
I like the idea. Merged ColdMethodsSystem installation into RegistrationModule, but I'd rather try merging Core and Registration separately